### DISCUSSION TITLE
预览pdf时出现问题

### DISCUSSION THREAD

**Discussion by LTcorpio** [AlistGo/alist#648] [Q&A]
服务器部署在阿里云上，当我尝试预览一个PDF时，只显示了我自定义的body的内容，其它内容都没有显示，无法正常预览。但是我本机的WSL调试时是可以正常预览pdf的。

挂载的是123云盘。还有，123盘也不能预览office嘛，按理来说它也是公网环境下的。

**Comment by xhofe** [comment]
pdf： #619
office： #594
123盘的下载链接并不是标准端口。
